com.ibm.wala.core/src/main/java/com/ibm/wala/core/util/strings/StringStuff.java

```diff
@@ -50,11 +50,7 @@ public static void padWithSpaces(StringBuilder b, int length) {
     if (b == null) {
       throw new IllegalArgumentException("b is null");
     }
-    if (b.length() < length) {
-      for (int i = b.length(); i < length; i++) {
-        b.append(' ');
-      }
-    }
+    b.append(" ".repeat(Math.max(0, length - b.length())));
```

Comment:
To me, it would be more readable to keep `if (b.length() < length) {` and put the `append()` call inside that condition than to use `Math.max()`

Reply:
Sure, will do.